web/docs/tutorial/07-auth.md

```diff
@@ -19,24 +19,16 @@ First, we'll create a Todo list for what needs to be done (luckily we have an ap
 
 ## Creating a User Entity
 
-Since Wasp manages authentication, it expects certain fields to exist on the `User` entity. Specifically, it expects a unique `username` field and a `password` field, both of which should be strings.
+Since Wasp manages authentication, it will create [the auth related entities](/docs/auth/entities) for us. However, we still need to add the `User` entity that will help us keep track of which user owns which tasks.
```

Comment:
```suggestion
Since Wasp manages authentication, it will create [the auth related entities](/docs/auth/entities) for us in the background, that we don't have to worry about. However, we still need to add the `User` entity that will help us keep track of which user owns which tasks.
```